internal/cypress/v1/config.go

```diff
@@ -261,10 +262,27 @@ func shardSuites(rootDir string, suites []Suite, ccy int) ([]Suite, error) {
 		if err != nil {
 			return shardedSuites, err
 		}
+
 		if len(files) == 0 {
 			msg.SuiteSplitNoMatch(s.Name, rootDir, s.Config.SpecPattern)
 			return []Suite{}, fmt.Errorf("suite '%s' patterns have no matching files", s.Name)
 		}
+
+		// TODO: Opt in to grep/grepTags filtering explicitly?
```

Comment:
Probably better/safer 😅

Reply:
👍 agreed. Now I just need to figure out where to put it and how to name it... 🤔